engine/common/api_helpers/custom_fields.py

```diff
@@ -102,7 +103,16 @@ def to_internal_value(self, data):
         if not request or not queryset:
             return None
 
-        return queryset.filter(organization=request.user.organization, public_primary_key__in=data).distinct()
+        users = queryset.filter(organization=request.user.organization, public_primary_key__in=data).distinct()
+        users_ppk = [u.public_primary_key for u in users]
+
+        if not self.db_verification:
+            return users
+
+        for d in data:
```

Comment:
Would this check be equivalent to do something like
`len(set(data)) == users.count()`
(ie. make sure we have a user per provided ppk)? I guess you could do a set difference to get the missing users too?

Reply:
I have Made the changes accordingly